pom.xml

```diff
@@ -596,6 +601,11 @@
                 <version>1.7.10</version>
             </dependency>
             <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-log4j12</artifactId>
+                <version>1.7.25</version>
```

Comment:
So you showed me the dependency tree and it looks like the caf-common overrides the version coming through slf4j-log4j12. It also looks to be happening to log4j, as it's pulling in 2.0 from caf-common instead of 2.8.2. 

Do we add a new dependency into caf-storage to ensure it uses the later version or should we try updating caf-common to later versions and see if they work with other projects as well?